src/tools/dashboard/components/Dashboard.tsx

```diff
@@ -34,6 +37,18 @@ const Dashboard = ({ inPanel }: { inPanel?: boolean }) => {
     }
   );
 
+  // All of this should probably part of the sliding panel logic
```

Comment:
Would be handy to have a TODO and jira reference for integrating these changes into franklin

Reply:
I'll create that Jira over the course of the day then 👍🏽